patches/@snapshot-labs+snapshot.js+0.3.56.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/node_modules/@snapshot-labs/snapshot.js/src/sign/index.ts b/node_modules/@snapshot-labs/snapshot.js/src/sign/index.ts
+index 6ac4a87..2cf9367 100644
+--- a/node_modules/@snapshot-labs/snapshot.js/src/sign/index.ts
++++ b/node_modules/@snapshot-labs/snapshot.js/src/sign/index.ts
+@@ -28,6 +28,7 @@ import {
+   aliasTypes
+ } from './types';
+ import hubs from '../hubs.json';
++import { _TypedDataEncoder } from '@ethersproject/hash';
```

Comment:
@midgerate why did we need this patch here? Isn't this the same as the PR @bonustrack opened just without this patch?

Reply:
Was just there for dubugging purposed. So that I could console the hash.